test/runtests.jl

```diff
@@ -909,18 +909,47 @@ else  # run HiGHS tests
         end
 
         @testset "Minimize Unserved Load" begin
-                
+            d = JSON.parsefile("./scenarios/outage.json")
+            d["ElectricLoad"]["loads_kw"] = ones(8760)*200.0
+            d["ElectricLoad"]["loads_kw"][5100:5109] .= 410.0
+            d["ElectricLoad"]["loads_kw"][5200:5209] .= 410.0
+            d["ElectricLoad"]["critical_load_fraction"] = 0.5
+            d["PV"]["existing_kw"] = 0.0
+            d["PV"]["min_kw"] = 100.0
+            d["PV"]["max_kw"] = 100.0
+            d["CHP"]["min_kw"] = 100.0
+            d["CHP"]["max_kw"] = 100.0
+            d["Generator"]["existing_kw"] = 0.0
+            d["Generator"]["min_kw"] = 100.0
+            d["Generator"]["max_kw"] = 100.0
+            d["ElectricStorage"]["min_kw"] = 20
+            d["ElectricStorage"]["max_kw"] = 20
+            d["ElectricStorage"]["min_kwh"] = 50
+            d["ElectricStorage"]["max_kwh"] = 50
+            d["Financial"]["microgrid_upgrade_cost_fraction"] = 0.0
+            s = Scenario(d)
```

Comment:
@zolanaj just curious, was there a compute time advantage to creating the Scenario and REoptInputs first, rather than just running `run_reopt(m,d)` ?

Reply:
I don't think there's any difference in computing - run_reopt(m, d) does the same sequence of commands.  I've always used the Scenario and REoptInputs creation upfront for debugging purposes; it's fewer lines of code in the testing though so I can convert it.

Reply:
done in 674780496cf626717396860c4257af90ada54ffd for the updated tests only; we can probably do this more widely but I don't see any reason for a computational difference so that's low priority.